playbooks/roles/common-apache2/tasks/main.yml

```diff
@@ -1,32 +1,59 @@
 ---
 # This role contains common tasks that will run on all nodes.
 
-- name: add `ppa:certbot/certbot` repo
-  become: yes
-  apt_repository:
-    repo: ppa:certbot/certbot
-
 - name: install common packages
-  become: yes
   apt:
     name: "{{ packages }}"
     update_cache: yes
   vars:
     packages:
     - apache2
     - build-essential
-    - certbot
     - git
     - libffi-dev
     - openssl
-    - python-certbot-apache
     - python-dev
     - python-pip
     - python-setuptools
 
+- name: Install snapd
+  apt:
+    name: snapd
```

Comment:
Can just be added to the list above

Reply:
I added it to the list initially, but later decided to move it to a distinct task to keep tasks that were "ported" over from the certbot role separated.